__tests__/app.test.js

```diff
@@ -424,4 +430,45 @@ describe("/api", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id (comment_count)", () => {
+    test("GET 200: Should respond with the comment count for an article array", () => {
+      return request(app)
+        .get("/api/articles/5")
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(Array.isArray(article)).toBe(true);
+          expect(article.length).toBe(1);
+          article.forEach((object) => {
+            expect(object).toHaveProperty("author");
+            expect(object).toHaveProperty("title");
+            expect(object).toHaveProperty("article_id");
+            expect(object).toHaveProperty("body");
+            expect(object).toHaveProperty("topic");
+            expect(object).toHaveProperty("created_at");
+            expect(object).toHaveProperty("votes");
+            expect(object).toHaveProperty("article_img_url");
```

Comment:
unsure that you need to check for all of these ^^ seeing as the test is just for comment_count :)